features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,21 @@
-# put step definitions in this file
\ No newline at end of file
+Given(/^I am on the home page$/) do
+  visit '/'
+end
+
+When(/^I visit the name page for "([^"]*)"$/) do |arg1|
+  visit '/name/Amber'
+end
```

Comment:
You are not using the variable `arg1` in this step definition.

Consider the following code:
```
When(/^I visit the name page for "([^"]*)"$/) do | name |
   visit "/name/#{name}"
 end
```